internal/server/handlers.go

```diff
@@ -41,7 +41,7 @@ func (a *API) CreateToken(c *gin.Context, r *api.EmptyRequest) (*api.CreateToken
 			return nil, fmt.Errorf("%w: failed to update identity info from provider: %s", internal.ErrUnauthorized, err)
 		}
 
-		token, err := access.CreateToken(rCtx)
+		token, err := data.CreateIdentityToken(rCtx.DBTxn, rCtx.Authenticated.User.ID)
```

Comment:
TODO: add a comment at the top of this function to say why this doesn't require authorization

Reply:
If it's clear this route only requires authentication, then we probably don't even need the TODO since all users can call this for now.